### DISCUSSION TITLE
How to get data after selecting emoji to parse later?

### DISCUSSION THREAD

**Discussion by Svencap** [missive/emoji-mart#718] [Q&A] [answered]
I can not figure out how to get the data of the emoji itself. Console.log is passed to the onEmojiSelect event in the documentation. Please tell me how to get this object if you pass your callback. 
For example 
```
import dataEmoji from '@emoji-mart/data';
import Picker from '@emoji-mart/react';

const handleClickEmoji = ( e ) => {
 const data = ?;
 //Next is something;
};

<Picker previewPosition="none" data={dataEmoji} onEmojiSelect={handleClickEmoji} />
```

**Comment by EtienneLem** [comment, marked as answer]
> Console.log is passed to the onEmojiSelect event in the documentation

This means that whatever arguments is sent to `onEmojiSelect`, they are being sent to `console.log`. In your example, `e` is the emoji object / data.

```js
const data = e
```

I would personally name that variable `emoji` instead of `e`.
